common/gamespy_message.go

```diff
@@ -13,7 +13,7 @@ type GameSpyCommand struct {
 }
 
 var (
-	InvalidGameSpyCommand = errors.New("invalid command received")
+	InvalidGameSpyCommand = errors.New("Invalid command received")
```

Comment:
The Go Style guide states that the error string should not start with a capital letter.

Reply:
Oh, I didn't know that

Reply:
I've fixed it now

Reply:
> The Go Style guide states that the error string should not start with a capital letter.

Thank you for catching that.